jobdsl/organisations-beta.groovy

```diff
@@ -11,12 +11,13 @@ List<Map> orgs = [
         [name: 'Adoption'],
         [name: 'AM'],
         [name: 'BSP', regex: '(send-letter-client|send-letter-service|send-letter-performance-tests|bulk-scan-.*|blob-router-service|reform-scan-.*)'],
-        [name: 'CDM', regex: '\\b(?:document-management-store-app|dm-shared-infrastructure|ccd.*)\\b'],
+        [name: 'CDM', regex: 'ccd.*'],
         [name: 'CMC'],
         [name: 'CNP'],
         [name: 'CTSC'],
         [name: 'DIV', displayName: "Divorce"],
         [name: 'ETHOS', displayName: "Ethos replacement"],
+        [name: 'EM',displayName: 'Evidence Management', regex: '\\b(?:document-management-store-app|dm-shared-infrastructure|em.*)\\b'],
```

Comment:
does it really need this fancy regex? something about a word boundary and a negative look around / ahead afaik ?

Reply:
just copied over from ccd , simplified it :)